hnn_core/viz.py

```diff
@@ -326,7 +336,7 @@ def plot_cells(net, ax=None, show=True):
 
 def plot_tfr_morlet(dpl, *, freqs, n_cycles=7., tmin=None, tmax=None,
                     layer='agg', decim=None, padding='zeros', ax=None,
-                    colorbar=True, show=True):
+                    colormap='inferno', colorbar=True, show=True):
```

Comment:
should this function also be a method of `Dipole` ?

Reply:
Sure! and `plot_psd` too.